pkg/lang/ir/python.go

```diff
@@ -24,6 +24,17 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+func (g Graph) GetAppropriatePythonVersion() (string, error) {
+	version := *g.Language.Version
+	if version == "3" || version == "" {
+		return defaultVersion, nil
+	}
+	if strings.HasPrefix(version, "3.") {
+		return version, nil
+	}
+	return "", errors.Errorf("python version %s is not supported", version)
```

Comment:
We do not support Python 2.x, does it work for you? @VoVAllen @kemingy

Reply:
I forget to discuss with others. But I do think it is not a good idea to support python2. >_<!

Reply:
LGTM

Reply:
LGTM. Python 2.x should be deprecated in all ways